modules/admin_manual/nav.adoc

```diff
@@ -99,16 +93,22 @@
 **** xref:configuration/server/harden_server.adoc[Harden Server]
 **** xref:configuration/server/import_ssl_cert.adoc[Import SSL Cert]
 **** xref:configuration/server/index_php_less_urls.adoc[Index PHP Less URLs]
+**** xref:configuration/server/occ_command.adoc[OCC Command]
 **** xref:configuration/server/language_configuration.adoc[Language Configuration]
 **** xref:configuration/server/legal_settings_configuration.adoc[Legal Settings Configuration]
 **** xref:configuration/server/logging/logging_configuration.adoc[Logging Configuration]
 ***** xref:configuration/server/logging/request_tracing.adoc[Request Tracing]
 ***** xref:configuration/server/logging/providing_logs_and_config_files.adoc[Retrieve Log Files and Configuration Settings]
-**** xref:configuration/server/oc_server_tuning.adoc[Server Tuning]
-**** xref:configuration/server/occ_command.adoc[OCC Command]
 **** xref:configuration/server/reverse_proxy_configuration.adoc[Reverse Proxy Configuration]
-**** xref:configuration/server/security_setup_warnings.adoc[Security Setup Warnings]
+**** xref:configuration/server/security/index.adoc[Security]
+***** xref:configuration/server/security/brute_force_protection.adoc[Brute-Force Protection]
+***** xref:configuration/server/security/oauth2.adoc[OAuth2]
+***** xref:configuration/server/security/password_policy.adoc[Password policy]
+***** xref:configuration/server/security_setup_warnings.adoc[Security Setup Warnings]
+***** xref:configuration/server/security/hsmdaemon/index.adoc[The HSM (Hardware Security Module) Daemon]
```

Comment:
Wouldn´t it look better to write:
`The HSM (Hardware Security Module) Daemon`--> `HSM (Hardware Security Module) Daemon`
and reorder it accordingly?

Reply:
Might do. No harm in changing it.

Reply:
Implemented.